Library/Formula/encfs.rb

```diff
@@ -2,10 +2,11 @@
 
 class Encfs < Formula
   homepage 'https://vgough.github.io/encfs/'
+  version "1.8.1"
```

Comment:
you can remove this, it will be detected from the URL:

```
Error: 1 problems in 1 formulae
encfs:
 * Stable: version 1.8.1 is redundant with version scanned from URL
```